content/blog/2023/07/05/research-forum-4-marc-santolini-liam-kavanagh.md

```diff
@@ -8,12 +8,17 @@ categories:
 image: /assets/images/research-forum-marc-santolini-liam-kavanagh-collaborative-citizen-science.jpg
 ---
 
-**Our next research forum call is on Friday 21st July at 3:00pm CET.**
+>This video forms part of our series of [[/research|research community]] monthly fora.
 
-[REGISTER HERE](https://us02web.zoom.us/meeting/register/tZAkduqvqzgqGN3Q2xytYcFB-emg2ocZ6W-H )
+In this call, Marc and Liam lead a discussion on how an extreme citizen science of contemplation can be built, bringing together the wisdom of ancient contemplative practices and the rigor of modern scientific methodologies.
 
-In this call, Marc and Liam will lead a discussion on how an extreme citizen science of contemplation can be built, bringing together the wisdom of ancient contemplative practices and the rigor of modern scientific methodologies.
+As Liam points out in the talk, 'Relative truths are the kinds of truths that we can grasp with the intellect, and ultimate truths are the kinds of things that you only see if you let go of intellectual understanding, and a large part of what a lot of what contemplatives puzzle over are these ultimate truths'. Given this distinction, how can contemplatives help guide science? Additionally, how can this be done in a collaborative and meaningful way?
```

Comment:
1. The opening of this could read better. In the talk Liam discusses... 
A key tenet of the conversation centred around ... 

2. "what a lot of what contemplatives puzzle over are these ultimate truths"

is the 2nd what needed?